Tests/TestFrameworks/NUnit4.Specs/FrameworkSpecs.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace NUnit4.Specs;
+
+[TestFixture]
+public class FrameworkSpecs
+{
+    [Test]
+    public void When_nunit4_is_used_it_should_throw_nunit_exceptions_for_assertion_failures()
```

Comment:
🔧 Please use the new namign convetion.

Reply:
Implemented in [0c4712f](https://github.com/fluentassertions/fluentassertions/pull/2483/commits/0c4712fa52b1cecaa9b6c74475bbcd33b8ff1faa).